include/ck/tensor_operation/gpu/element/unary_element_wise_operation.hpp

```diff
@@ -7,12 +7,170 @@
 #include "ck/utility/math.hpp"
 #include "ck/utility/math_v2.hpp"
 #include "ck/utility/type_convert.hpp"
+#include "ck/utility/amd_inline_asm.hpp"
 #include <cassert>
 
 namespace ck {
+
+// https://github.com/NVIDIA/FasterTransformer/blob/main/src/fastertransformer/cutlass_extensions/include/cutlass_extensions/interleaved_numeric_conversion.h
```

Comment:
@zjing14 @illsilin I'm not sure we can just use that code like that. Nvidia has BSD license which would obligate us add copyright notice.